Modules/Core/Common/include/itkBSplineInterpolationWeightFunction.h

```diff
@@ -87,6 +87,7 @@ class ITK_TEMPLATE_EXPORT BSplineInterpolationWeightFunction
 
   /** The support region size: a hypercube of length SplineOrder + 1 */
   static constexpr SizeType SupportSize{ SizeType::Filled(VSplineOrder + 1) };
+  // Declaration here, definition must be outside of class until C++17, see .hxx file for linker definition
```

Comment:
Thanks very much for addressing this problem @hjmjohnson I did not realize that my pull request #2711 would cause those link errors, sorry!